packages/transactions/src/OutboundShipment/DetailView.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react';
+import { useQueryClient } from 'react-query';
+import { useNavigate, useParams } from 'react-router';
+import { request, Transaction } from '@openmsupply-client/common';
+import { getMutation, getDetailQuery } from '../api';
+import { createDraftStore, useDraftDocument } from '../useDraftDocument';
+
+const queryFn = (id: string) => async (): Promise<Transaction> => {
+  const result = await request('http://localhost:4000', getDetailQuery(), {
+    id,
+  });
+  const { transaction } = result;
+  return transaction;
+};
+
+const mutationFn = async (updated: Transaction): Promise<Transaction> => {
+  const patch = { transactionPatch: updated };
+  const result = await request('http://localhost:4000', getMutation(), patch);
+  const { upsertTransaction } = result;
+  return upsertTransaction;
+};
+
+const placeholderTransaction: Transaction = {
+  customer: '',
+  total: '',
+  date: '',
+  supplier: '',
+};
```

Comment:
Found this to be the nicest way to do the new thing.. seed with this placeholder and that becomes the `data` value returned from `useQuery`.. so when it's a `new` outbound, use this placeholder, otherwise don't pass one. Then everything else is the same as editing, kinda